python/hail/api1/dataset.py

```diff
@@ -1161,12 +1161,24 @@ def concordance(self, right):
 
         **Notes**
 
-        This method computes the genotype call concordance between two bialellic variant datasets.
-        It performs an inner join on samples (only samples in both datasets will be considered), and an outer join
-        on variants. If a variant is only in one dataset, then each genotype is treated as "no data" in the other.
-        This method returns a tuple of three objects: a nested list of list of int with global concordance
-        summary statistics, a key table with sample concordance statistics, and a key table with variant concordance
-        statistics.
+        This method computes the genotype call concordance between two
+        bialellic variant datasets.  It requires unique sample IDs and
```

Comment:
typo in ballelic.

Reply:
I think biallelic is the hardest word to type in the English language.

Reply:
aggregable would like a word